main.go

```diff
@@ -115,6 +116,9 @@ func main() {
 
 	tenantService := tenant.NewDBService(db)
 
+	// Check & do all tenants update
+	go update.NewTenantsUpdater(10*time.Minute, db, config, authService, clusterService, controller.OSUpdater{}).UpdateAllTenants()
```

Comment:
Maybe worth moving the delay/timeout to configuration?

Reply:
moved here 006ee59